pandas/core/window/indexers.py

```diff
@@ -113,8 +113,17 @@ def get_window_bounds(
         closed: Optional[str] = None,
     ) -> Tuple[np.ndarray, np.ndarray]:
 
+        # error: Argument 4 to "calculate_variable_window_bounds" has incompatible
+        # type "Optional[bool]"; expected "bool"
+        # error: Argument 6 to "calculate_variable_window_bounds" has incompatible
+        # type "Optional[ndarray]"; expected "ndarray"
         return calculate_variable_window_bounds(
-            num_values, self.window_size, min_periods, center, closed, self.index_array
+            num_values,
+            self.window_size,
+            min_periods,
+            center,  # type: ignore[arg-type]
```

Comment:
so center could be None according to the docstring, but once passed on to cython would be interpreted as False?

Reply:
my impression is that the annotation here is wrong, which im putting an ignore on for now